### PR TITLE
Adding tests for submitter

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
Backfilling tests for submitter

### How to review 
Tests should pass

### CODE REVIEW

File: tests/app/submitter/test_submitter.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    def test_send_message_fails(self):
+        # Given
+        submitter = RabbitMQSubmitter()
+
+        # When
+        published = submitter.send_message(message={}, queue='test_queue')
+
+        # Then
+        self.assertFalse(published, 'send_message should fail to publish message')
```
Comment (jonnyshaw89) [inline tests/app/submitter/test_submitter.py]: Why should it fail? test doesn't explain why. so if this test fails i wouldn't know what was wrong

Comment (jonnyshaw89) [inline tests/app/submitter/test_submitter.py:20]: Why should it fail? test doesn't explain why. so if this test fails i wouldn't know what was wrong

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/app/submitter/test_submitter.py (hunk 1 above)
Comment (benjefferies) [reply tests/app/submitter/test_submitter.py:20]: Could you take a look again and let me know if you approve?

Comment (jonnyshaw89) [reply tests/app/submitter/test_submitter.py:20]: Beautiful

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**